client/src/document/edit.tsx

```diff
@@ -65,11 +65,11 @@ function EditForm({ data, url }) {
     setAutoSaveEnabled(!autosaveEnabled);
   }
 
-  const putDocumentDebounced = useRef(debounce(1000, putDocument));
```

Comment:
useCallback could've been an option here, but since basically every render would invalidate the memoization anyway, I just went without. I don't see a use-case for useRef here.